src/config/dynamic-theme-fixes.config

```diff
@@ -17100,6 +17100,64 @@ CSS
 
 ================================
 
+upwork.com
+/*---------------Home page fixes-------------------*/
+/* fix 1 Remove drop shadow of green button in right column */
+
+CSS
+.profile-completeness-nudges-tiles-alternative .carousel-wrapper .up-icon svg {
+ filter: drop-shadow(0 0 0px var(--white)) drop-shadow(0 0 0px var(--white)); 
+}
+/* ------fix 2 REMOVE WHITE GRADIENT HIGHLIGHTS------ */
+.up-skill-container .up-btn.up-btn-next::before { opacity: 0; }
+.up-skill-container .up-btn.up-btn-prev::before { opacity: 0; }
+.up-tab-scroll-hint:after { background: none;
+}
+/*-----FIX 3 skeleton loading messages page------*/
+.up-d-skeleton {
+    color: transparent !important;
+    background-color: var(--darkreader-bg--skeleton-color)  !important;
+    background-image: linear-gradient(90deg,var(--darkreader-bg--skeleton-color),#24242400,var(--darkreader-bg--skeleton-color))  !important;
+}
+/*-----Fix 4 Background color missmatch------*/
+.layout>.container, .layout>.container-visitor, .layout>main>.container, .layout>main>.container-visitor {
+   
+    background-color: #1f2223;
+    
+}
+.layout-container {
+    
+    background-color: #1f2223;
+}
+.layout-container {
+    
+    height: 430%;
+}
```

Comment:
This seems like a cosmetic fix, so it doesn't fix a faulty behavior from Dark Reader. But rather a opinionated fix.

Reply:
Removed it

Reply:
Are there more of such fixes? I'm not really able to tell, some seem to be indeed fixing faulty behavior, but other just seems like personal preferences.

Reply:
Hold on. I'll remove some stuff now. There was some things I was suppose to remove but forgot